src/hooks/useSearchCurrencyList.ts

```diff
@@ -217,19 +213,18 @@ const useSearchCurrencyList = (searchQuery: string, searchChainId = ChainId.main
 
   const getCrosschainVerifiedAssetsForNetwork = useCallback(
     async (chainId: ChainId) => {
-      const fromChainId = inputChainId !== chainId ? inputChainId : '';
       const results = await searchCurrencyList({
         searchList: 'verifiedAssets',
         query: '',
         chainId,
-        fromChainId,
+        fromChainId: '',
```

Comment:
looks like `fromChainId` is not even used in the `searchCurrencyList` function, so we can remove it?

Reply:
same for other instances of `searchCurrencyList` in this file